src/EnrollTotpController.js

```diff
@@ -95,21 +93,15 @@ function (Okta, FactorUtil, FormController, FormType,
 
       formChildren: function () {
         var inputOptions = {
-          APPLE: '',
-          ANDROID: ''
+          APPLE: 'Apple',
```

Comment:
i guess no need to localize? language might be different though..

Reply:
Not sure. @mauriciocastillosilva-okta any suggestions ?